src/Migrations/add_arguments.hack

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
```

Comment:
Perhaps this should be in __Private. Don't feel strongly

Reply:
I put `whitespace_from_nodelist` in `__Private`, but `add_arguments` and `prepend_statement` seemed more generally useful so I put them here.

I'd argue that these functions have more right to be in a public namespace than all the migration/lint/etc. classes :)